lw1551/Power of Thor intro.cs

```diff
@@ -0,0 +1,85 @@
+// * Use *if statements* to check for thor’s updated location compared to the orb, then use *else statements* to compare them for each direction that thor can move in.
+// 
+// .. image:: Power_Of_Thor.png
+//
+// The idea is that Thor has an x and y position that is separate from the x and y position of the power, and this image shows that the plane is 40 wide by 18 high, but due to arrays this runs from 0 to 39 and 0 to 17.
+//
+// -------------------
+// Desgin Constraints
+// -------------------
+// 1. Thor cannot move beyond the coordinate plane defined by the game, which is **40 width by 18 height**.
+//
+// 2. Thor cannot run out of energy; the game allows for a **maximum of 100 energy**.
+//
+// ---------------------
+// Defined Variables
+// ---------------------
+//
+        string[] inputs = Console.ReadLine().Split(' ');
+        int lightX = int.Parse(inputs[0]); 
+        // The X position of the light of power
+        int lightY = int.Parse(inputs[1]); 
+        // The Y position of the light of power
+        int initialTX = int.Parse(inputs[2]); 
+        // Thor's starting X position
+        int initialTY = int.Parse(inputs[3]); 
+        // Thor's starting Y position
+ //
+        // The first two variables, lightX and lightY, are used to set where the coordinates of the light orb's location for each new level of the game. I then defined initialTX and initialTY to be Thor's initial location. When first designing this program, I had two addtional variables that would update to thor's new location, but I realized when working through this that the initialTX and initialTY variables can be used for both functions.
+//
+//
+//
+        string directionX = "";
+        // Blank X direction string to fill with direction thor will move in.
+        string directionY = "";
+        // Blank Y direction string to fill with direction thor will move in.
+        // These variables are used to read in the cardinal direction given by the player of the game.
+
+// ****************
+// Solution
+// ****************
+// This code operates by making multiple comparisons each time Thor moves. The loop begins by establishing that thor has a X and a Y direction, and creating a string variable that will store the Direction in it. For example, if Thor needs to move to the bottom right, then the if statements will check for whether Thor's X location is larger than the light's X value. The same will be done for the Y values of the game. Therefore after several runs through the code you will be moving in all cardinal directions. So the basic idea is that the strings will tell which cardinal direction we want to go in, while the code itself will run comparisons between thor's orginal/updated location and the light's location to decide if the move is valid for the game to be won. 
```

Comment:
Show a map / image / diagram that explains how compass directions relate to x and y coordinates.